docs/admin/etcd.md

```diff
@@ -1,49 +1,196 @@
 ---
 assignees:
-- lavalamp
-title: Configuring Kubernetes Use of etcd
+- mml
+- wojtek-t
+title: Upgrading and Rolling Back etcd
 ---
 
+## About etcd
+
 [etcd](https://coreos.com/etcd/docs/latest/) is a highly-available key value
 store which Kubernetes uses for persistent storage of all of its REST API
-objects.
+objects.  For the mechanics behind how kubernetes builds, distributes and
+deploys etcd, see _some doc_.
```

Comment:
Is _some doc_ just a placeholder for a link? If so, this needs to put in before we can Docs LGTM.